vnext/Desktop/WebSocketModule.cpp

```diff
@@ -59,15 +64,15 @@ std::vector<facebook::xplat::module::CxxModule::Method> WebSocketModule::getMeth
           }
         }
 
-        this->GetWebSocket(jsArgAsInt(args, 3), jsArgAsString(args, 0))->Connect(protocols, options);
+        this->GetOrCreateWebSocket(jsArgAsInt(args, 3), jsArgAsString(args, 0))->Connect(protocols, options);
```

Comment:
[nit] Doesn't look like an addition of comments :)

Reply:
Agreed, last-minute suggestion by @Khalef1.
It is a trivial change and ultimately impacts code readibility, which is the one of the purposes of this PR.